pkgs/applications/misc/chrysalis/default.nix

```diff
@@ -1,35 +1,42 @@
 { lib, appimageTools, fetchurl }:
 
 let
+  owner = "keyboardio";
```

Comment:
not sure I see the point of moving `owner` out here, probably should be left as is.

Reply:
fixed.